.ci/install_crio.sh

```diff
@@ -16,8 +16,27 @@
 
```

Comment:
this script must be compatible with all other distros, if not then it should be part of ```setup_env_ubuntu.sh```